clients/extension/styles/lighthouse.css

```
@@ -43,168 +53,127 @@ html, body {
+  margin: 8px auto;
+  padding: 6px 14px 6px 14px;
+  border-radius: 13px;
+  background-color: var(--color-blue);
+  transition: 0.1s all;
 }
-
-.header-titles__throttling {
-  position: relative;
-  margin-top: 5px;
-  padding-left: 20px;
+.button--generate:not(:disabled):hover {
+  background-color: #042790;
+  border: 1px solid #0535C1;
 }
 
-.header-titles__throttling input {
-  position: absolute;
-  left: 0;
+.section {
+  display: grid;
+  justify-content: center;
 }
 
-.main {
-  height: 70px;
-  display: flex;
-  padding: 16px;
-  background-color: var(--lh-blue-color);
-  justify-content: space-around;
-  align-items: flex-end;
+.section:first-child {
+  border-bottom: solid 1px lightgray;
+  padding-bottom: 10px;
 }
 
-.button {
-  cursor: pointer;
-  font-family: 'Roboto', Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
+.devtools-header {
+  font-size: var(--font-size);
   font-weight: 500;
-  font-size: 12px;
-  border: none;
-  color: #fff;
-  background-color: #2979ff;
-  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
-  border-radius: 2px;
-  padding: 7px 12px 5px 12px
+  margin-bottom: 5px;
 }
 
-.button--generate {
-  font-weight: 500;
-  font-size: 16px;
-  padding: 8px 16px 10px 16px;
-  align-self: stretched;
-  margin-left: auto;
+.devtools-note {
+  font-size: 14px;
+  color: grey;
 }
 
-.button--configure {
-  margin-right: auto;
+.devtools-shortcut {
+  font-family: var(--report-font-family-monospace);
 }
 
-.button--ok {
-  margin: 0 auto;
+.button--configure {
+  position: absolute;
+  top: 15px;
+  right: 10px;
+  color: rgb(71, 71, 71);
+  cursor: pointer;
 }
-
-.button--report-error {
-  background-color: rgba(239, 83, 80, 0.81);
-  text-decoration: none;
-  padding: 3px 5px;
-  font-size: .8em;
-  text-align: center;
-  display: block;
-  margin-top: 5px;
+.show-options .button--configure {
+  color: var(--color-blue);
+}
+.button--configure.disabled {
+  color: grey;
+  cursor: not-allowed;
 }
 
-.subpage {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background-color: var(--lh-bg-color);
-  pointer-events: none;
-  display: flex;
-  flex-direction: column;
-  opacity: 0;
-  will-change: opacity;
-  transition: opacity 0.150s cubic-bezier(0,0,0.41,1);
+.psi-disclaimer {
+  font-size: 12px;
   text-align: center;
-  padding: 16px;
+  color: grey;
 }
 
-.subpage--visible {
-  opacity: 1;
-  pointer-events: auto;
+.psi-disclaimer a {
+  color: inherit;
+  text-decoration: none;
 }
 
-.status {
-  align-items: center;
-  justify-content: flex-start;
-  background-color: #fff;
+.options__title {
+  font-size: var(--font-size);
+  font-weight: 500;
+  margin-bottom: 5px;
 }
 
-.subpage--visible.status .lighthouse-effects__wrapper {
-  animation-play-state: running;
+.section--options label {
+  display: flex;
+  align-items: center;
 }
 
-.status__msg {
-  font-size: 16px;
-  margin-top: 16px;
+.section--options input {
+  margin-top: 0;
+  margin-bottom: 0;
```

```suggestion
  margin-bottom: 0;
  margin-right: 8px;
```

gives some nice space between the checkbox and label